101-vm-with-powerbi-data-gateway/azuredeploy.json

```diff
@@ -0,0 +1,301 @@
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Premium_LRS",
+        "Standard_LRS"
+      ],
+      "metadata": {
+        "description": "Which type of storage you want to use"
+      }
+    },
+    "vmName": {
+      "type": "string",
+      "defaultValue": "PBIGateway",
+      "metadata": {
+        "description": "Local name for the VM can be whatever you want"
+      }
+    },
+    "vmAdminUserName": {
+      "type": "string",
+      "metadata": {
+        "description": "VM admin user name"
+      }
+    },
+    "vmAdminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password on VM"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "metadata": {
+        "description": "Desired Size of the VM. Any valid option accepted but if you choose premium storage type you must choose a DS class VM size."
+      },
+      "defaultValue": "Standard_DS2"
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "defaultValue": "[concat('vm-', uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "DNS Label for the Public IP. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "aadAppId": {
+      "type": "string",
+      "metadata": {
+        "description": "AAD Application Id"
+      }
+    },
+    "aadAppSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "AAD Application Secret"
+      }
+    },
+    "tenantId": {
+      "type": "string",
+      "defaultValue": "[subscription().tenantId]",
+      "metadata": {
+        "description": "AAD Tenant Id"
+      }
+    },
+    "gatewayName": {
+      "type": "string",
+      "metadata": {
+        "description": "Power BI Data Gateway name must be unique in subscription"
+      }
+    },
+    "gatewayRegionKey": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Power BI Data Gateway RegionKey eg: northeurope"
+      }
+    },
+    "gatewayRecovery": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Power BI Data Gateway recovery key"
+      }
+    },
+    "gatewayAdminIds": {
+      "type": "string",
+      "metadata": {
+        "description": "Power BI Data Gateway Administrator Ids (comma separted list of AAD Object Ids)"
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-with-powerbi-data-gateway/"
```

Comment:
move this above metadata - won't comment on the rest

Reply:
Done